stage0/entrypoint.go

```diff
@@ -26,9 +26,9 @@ import (
 )
 
 const (
-	enterEntrypoint = "coreos.com/rkt/stage1/enter"
-	runEntrypoint   = "coreos.com/rkt/stage1/run"
-	gcEntrypoint    = "coreos.com/rkt/stage1/gc"
+	enterEntrypoint = "stage1-enter"
+	runEntrypoint   = "stage1-run"
+	gcEntrypoint    = "stage1-gc"
```

Comment:
@alban Seems this is a support for us to use ACIdentifier here... /cc @jonboulle